SPECS/mstflint/mstflint.spec

```diff
@@ -40,13 +56,33 @@ find . -type f -iname '*.cpp' -exec chmod a-x '{}' ';'
 %configure --enable-fw-mgr --enable-adb-generic-tools
 %make_build
 
+# Compile mstflint driver
+export EXTRA_CFLAGS='-DVERSION=\"%version\"'
+cd kernel
+set -- *
+mkdir source
+mv "$@" source/
+mkdir obj
+
+rm -rf obj
```

Comment:
Why create a directory and immediately remove it?

Reply:
Removed.